fix failed install of conf files in /etc/vdr
reported by several users
patch provided  by negril @ IRC #gentoo-vdr
diff -Naur vdr-1.7.30.orig/Makefile vdr-1.7.30/Makefile
--- vdr-1.7.30.orig/Makefile	2012-09-27 19:10:50.614100146 +0200
+++ vdr-1.7.30/Makefile	2012-09-27 19:12:22.480863962 +0200
@@ -223,11 +223,11 @@
 
 install-dirs:
 	@mkdir -p $(DESTDIR)$(VIDEODIR)
-	@mkdir -p $(DESTDIR)$(CONFDIRDEF)
 	@mkdir -p $(DESTDIR)$(CACHEDIRDEF)
 	@mkdir -p $(DESTDIR)$(RESDIRDEF)
 
 install-conf:
+	@mkdir -p $(DESTDIR)$(CONFDIRDEF)
 	@cp *.conf $(DESTDIR)$(CONFDIRDEF)
 
 
